mobile_config/src/client/mod.rs

```diff
@@ -29,6 +31,8 @@ pub enum ClientError {
     UnknownServiceProvider(String),
     #[error("Invalid GatewayInfo proto response {0}")]
     InvalidGatewayInfoProto(#[from] crate::gateway_info::GatewayInfoProtoParseError),
+    #[error("error {0}")]
+    AnyhowError(#[from] anyhow::Error),
```

Comment:
This was added to catch errors returned from `file_store::traits::TimestampDecode` which returns a `file_store::Error`. We already provide a mapping to that above with `VerificationError`, though it's not used anywhere. 

Maybe we can rename that 
```rs
    #[error("file_store: {0}")]
    FileStore(#[from] file_store::Error),
```
Then we'll get a full trace of what went wrong. It'll be a bit more verbose, but it removes the option to rely on string errors.

Reply:
addressed here: e8643f61bafa27c8c1a9680da8f625f068f01393